content/06.quality_criteria.md

```diff
@@ -78,6 +78,15 @@ programming language being used.
 4. Code style compliance testing MUST be automated and MUST be triggered for
    each candidate change in the source code.
 
+### Code metadata
+Metadata for the software component, is a way to facilitate the citation of
+that component [@url:http://dx.doi.org/10.7717/peerj-cs.86] and to have assigned
+a Digital Object Identifier (DOI). Is is also key
+towards preservation, discovery, reuse, and attribution of the software component.
+
+A metadata file SHOULD exist along side the code, under its Version Control System (VCS).
+The metadata file SHOULD be updated when needed, as is the case of a new version.
+
```

Comment:
We can leave the discussion of having DOIs for each relevant version of the software for `v4.0` of the baseline

Reply:
this part is not about the DOI necessarily, other metadata may need to be updated, not modifying this for now

Reply:
not DONE, but up for further discussion